src/main/resources/hudson/plugins/jira/JiraSite/config.jelly

```diff
@@ -24,6 +24,9 @@
   <f:entry>
     <f:checkbox title="${%Update Relevant Jira Issues For All Build Results}" field="updateJiraIssueForAllStatus" />
   </f:entry>
+  <f:entry>
+     <f:checkbox title="${%Use Bearer authentication}" field="useBearerAuth" />
```

Comment:
I wonder, does it make sense to have both auth options: `useHTTPAuth` and `useBearerAuth` as checkbox? What happens if someone fills in both? 
If it does, I'd suggest moving those checkboxes to be together in the UI.
If it doesn't, then you can use Radio/option?

Reply:
I wasn't able to find where `useHTTPAuth ` is used 🤔

Reply:
@rantoniuk as I supposed, this variable it's no more used. It was introduced in this commit https://github.com/jenkinsci/jira-plugin/commit/20e5c7742b0a53c16a99d66cd6aec3e03a60a038 and used inside `createSession` function. Now, this part of code is no more present (https://github.com/jenkinsci/jira-plugin/blob/b061f3fee1e3f99aa02b04b842aa98eadc4fd328/src/main/java/hudson/plugins/jira/JiraSite.java#L536-L564), it has been removed here https://github.com/jenkinsci/jira-plugin/commit/85c6b2aa8d8bdd34973d5f46b74b1c2eb3f131a4.

Reply:
What  do you think if I move `useHTTPAuth` and `useBearerAuth` in a list with default value to HTTP? Remove this variable means touch too much code IMO. Maybe it should be done in another PR

Reply:
In the screenshots you can see an example. However, this implementation need further changes since the variable of the select dropdown list should be saved in the `JiraSite` builder and current implementation will change. IDK if the best solution, for now, is to just remove `useHTTPAuth` from `config.jelly` and then open a new PR by removing all code related to `useHTTPAuth` variable.
![Screenshot 2023-03-30 at 15 38 01](https://user-images.githubusercontent.com/106666672/228854638-d796a4b7-23a9-4138-953d-e739536dba9a.png)
![Screenshot 2023-03-30 at 15 37 58](https://user-images.githubusercontent.com/106666672/228854741-f28e4c30-1934-4448-b964-54ee570f767b.png)

Reply:
I hid temporarily `useHTTPAuth` checkbox. Then, if needed, I can open a new PR and cleanup the code for `useHTTPAuth` variable 👍

Reply:
Thanks a lot for this contribution, you did a great job! Fingers crossed 🤞🏻 that you find some time to help with the clean up in the future!

Reply:
Yeah that could be done for sure 👍 In the meantime, I've another question posted [here](https://github.com/jenkinsci/jira-plugin/pull/145#issuecomment-1500275153)

Reply:
Oh, I missed this! Is this blocking this PR from being merged? I don't see `addAttachment` anywhere in this PR, but maybe I missed it somewhere. If that's the case, is there a test that can be added to validate this?

Reply:
No, it's not blocking this PR to be merged 👍 I'll provide you some error logs asap

Reply:
So if I understand correctly this is an unrelated topic, please open a separate issue for this one.

Reply:
Yeah correct, I just asked on this old PR since I thought that someone already know the response to my question 🤣 I'll open a new issue